frontend/src/pages/Graph/SummaryPanelEdge.tsx

```diff
@@ -169,95 +174,121 @@ export class SummaryPanelEdge extends React.Component<SummaryPanelPropType, Summ
         </div>
       );
     };
+    const ambientBadge = waypoint ? <AmbientBadge tooltip={'reported from Ambient'}></AmbientBadge> : undefined;
+    const MainSummary = ({ waypointEdge }: { waypointEdge?: boolean }): React.ReactElement => {
+      const source = waypointEdge ? destData : sourceData;
+      const dest = waypointEdge ? sourceData : destData;
+      return (
+        <div>
+          <div className={panelHeadingStyle}>
+            {getTitle(`Edge (${prettyProtocol(protocol)})`, ambientBadge)}
```

Comment:
Yes, I was thinking this was not very consistent, I will remove it. 
But, I will think it would be useful to add somehow information about where the telemetry was reported (Like ztunnel or waypoint?)

Reply:
It may be useful, especially in a mixed proxy/ambient impl.  But if we are going to add that reporting we probably will have to add it all in one PR, and I think we can wait on it.